Tinylog/View Controllers/Tasks/TasksViewController.swift

```diff
@@ -93,9 +93,12 @@
         addTaskView.delegate = self
         return addTaskView
     }()
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/c61f36452fa17d076822a3b73830cc49c60091be/Tinylog/View Controllers/Tasks/TasksViewController.swift#L96"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TasksViewController.swift:96</code></td>
    </tr>
  </tbody>
</table>